src/either_or_both.rs

```diff
@@ -140,7 +140,7 @@ impl<A, B> EitherOrBoth<A, B> {
         }
     }
 
-    /// Apply the function `f` on the value `b` in `Right(b)` or `Both(a, _)` variants if it is
+    /// Apply the function `f` on the value `b` in `Left(b)` or `Both(a, _)` variants if it is
```

Comment:
Shouldn't this be ".. or `Both(b, _)`? And maybe it should use the placeholder `a`?

Reply:
Yeah I think we're `Both` `Right`, that'd `b` `a` better fix.  Updated!